CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
 
+## [Zebra 1.0.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0) - 2023-06-14
+
+This is our 1.0.0 stable release.
```

Comment:
> TODO: a summary of the significant user-visible changes in the release

#6888 seems like a critical fix for some users
#6908 and the Dockerhub images at the `latest` tag might be worth calling attention to

Reply:
#6888 is in there: https://github.com/ZcashFoundation/zebra/pull/6877/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR26

Reply:
I'd like to make sure users know about these changes, by putting them in the release summary at the top of the changelog.

Here's an example from a previous release:
https://github.com/ZcashFoundation/zebra/blob/d2a29b655257e3ac947199098ccc85af50cca92f/CHANGELOG.md?plain=1#L9-12

Reply:
I added 4b37f5a but wordsmithing welcome